specification/2.0/Specification.adoc

```diff
@@ -134,16 +134,103 @@ Some language in the specification is purely informative, intended to give backg
 
 If an entire chapter or section contains only informative language, its title is suffixed with "`(Informative)`". If not designated as informative, all chapters, sections, and appendices in this document are normative.
 
-
-
-
 All Notes, Implementation notes, and Examples are purely informative.
 
 
 [[introduction-technical-terminology]]
 === Technical Terminology
 
-The glTF Specification makes use of common engineering and graphics terms such as *Vertex*, *Normal*, *Texture*, etc. to identify and describe certain GPU constructs and their attributes, states, and behaviors.
+The glTF Specification makes use of linear algebra terms such as *axis*, *matrix*, *vector*, etc. to identify certain math constructs and their behaviors as defined in the <<iev,International Electrotechnical Vocabulary>>.
+
+The glTF Specification makes use of common engineering and graphics terms such as *image*, *buffer*, *texture*, etc. to identify and describe certain _glTF_ constructs and their attributes, states, and behaviors. This section defines the basic meanings of these terms in the context of the Specification. The Specification text provides fuller definitions of the terms and elaborates, extend, or clarify the definitions. When a term defined in this section is used in normative language within the Specification, the definitions within the Specification govern and supersede any meanings the terms may have in other technical contexts (i.e. outside the Specification).
+
+accessor::
+An object describing the number and the format of data elements stored in a binary buffer.
```

Comment:
Is this supposed to say buffer view?

```suggestion
An object describing the number and the format of data elements stored in a binary buffer view.
```

Reply:
I'm not sure about this since the buffer view itself does not store anything, it's just a sub-range of a buffer.